justfile

```diff
@@ -81,6 +81,19 @@ run *ARGS:
 bench *ARGS:
   cargo bench --benches {{ARGS}} -- --nocapture
 
+mkconfig_local NUM_NODES *ARGS:
+  cargo run --bin mkconfig -- -n {{NUM_NODES}} --sailfish-base-addr "127.0.0.1:8000" --decrypt-base-addr "127.0.0.1:10000" --producer-base-addr "127.0.0.1:11000" --mode "increment-port" {{ARGS}} | jq
+
+mkconfig_docker NUM_NODES *ARGS:
+  cargo run --bin mkconfig -- -n {{NUM_NODES}} --sailfish-base-addr "172.20.0.2:8000" --decrypt-base-addr "172.20.0.2:8001" --producer-base-addr "172.20.0.2:8002" --mode "increment-address" {{ARGS}} | jq
+
+mkconfig_cloud_single:
+  cargo run --bin mkconfig -- -n 5 \
+  --sailfish-base-addr "timeboost-lb-50d346419d44f480.elb.us-east-2.amazonaws.com:8000" \
+  --decrypt-base-addr "timeboost-lb-50d346419d44f480.elb.us-east-2.amazonaws.com:10000" \
+  --producer-base-addr "timeboost-lb-50d346419d44f480.elb.us-east-2.amazonaws.com:11000" \
```

Comment:
These are very specific. Not sure if they should be in here.

Reply:
True, do you have ideas how to properly generate these? 

cc @jparr721

Reply:
to i'll revert changes cloud configs, and we can update these in a separate PR.